start_docker.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 docker run -it \
   -p 8888:8888 \
+  -v $PWD/segmentation:/usr/local/lib/python3.6/site-packages/segmentation \
   -v $PWD/scripts:/scripts \
   -v $PWD/data:/data \
-  $USER/segmentation:latest
+  segmentation:latest
```

Comment:
I had someone come to me with a use case where they needed to upload a local file, which is not supported by this .sh script (needed an additional -v flag for that). In the future, do we expect more use cases like this, or can we just safely assume that this is all the user needs for the most part?

Reply:
I think for now it makes sense to have all of the notebooks that someone wants to run be located within the scope of the docker, but if it turns into a repeated issue we can certainly address that.

Reply:
I think adding a blurb to the readme to the effect that all relevant data/files/etc should be placed in the `data` folder should be sufficient.